crates/ssz_codegen/tests/input/test_serde_derives.ssz

```diff
@@ -0,0 +1,12 @@
+Slot = uint64
+
+### Test that serde derives add proper imports (mirrors identifiers use case)
+#~# derive: Copy, Hash, Serialize, Deserialize
```

Comment:
thinking now, we should probably change `Serialize, Deserialize` to `serde::Serialize, serde::Deserialize`
for future proofing `ssz-gen`.